docs/source/internals.rst

```diff
@@ -0,0 +1,111 @@
+.. module:: jupyter_contrib_nbextensions.nbconvert_support
+
+Notebook extension structure
+============================
+
+The nbextensions are stored each as a separate subdirectory of
+`src/jupyter_contrib_nbextensions/nbextensions`
```

Comment:
in rst, inline literals may need two backticks? I believe the notebook repo has somehow enabled use of single backticks, but I'm not sure how. The only reference to single backticks that I can find is [this](http://www.sphinx-doc.org/en/1.4.9/markup/inline.html), which seems to suggest that by default, single backticks have no special meaning